### PR TITLE
refactor: update text for more information

### PR DESCRIPTION
**Pull request by s-bessey**
Could use some advice on styling for the more info section. I thought about making the "I can..." parts somewhat larger, but didn't know the best way to do so (I thought of a couple ways I *could* but wasn't sure if it was the best way.)

### CODE REVIEW

**Review by eldu — COMMENTED** [review]
Adding a quick review before lunch leading into an afternoon of meetings. 

Perhaps we can add some more styling to the links? An underline on hover?

And maybe consider styling what you have as link right now as bolded text? And instead all the link to the document at the end of the bullet point?

Also let's open a new tab when a link is clicked! Maybe we need a link baby component for styling for a link. I think we have to set the target to blank?

And here is an icon for a new tab
https://fontawesome.com/icons/up-right-from-square?s=solid

**Comment by eldu** [comment]
For example, maybe something like this


> You can **educate your community about vaccines and build** trust. The following document has videos and messages you can share with people in your community. [Vaccine Outreach:
Community Partner Activation](https://health.ri.gov/publications/actionplans/COVID-vaccine-outreach.pdf)

**Comment by mcmcgrath13** [comment]
> Perhaps we can add some more styling to the links? An underline on hover?
> 
> And maybe consider styling what you have as link right now as bolded text? And instead all the link to the document at the end of the bullet point?

The link styling is the default with the `.content` class, so any changes should be made globally to that class and not just to this section I think.

File: src/views/AboutPage.vue [hunk 1]
```diff
@@ -27,25 +27,36 @@
...
+            >, and
+            <ExternalLink href="https://www.titanmodel.org/"
+              >TITAN </ExternalLink
+            >. For more about the People, Place, and Health Collective,
+            <ExternalLink href="https://medium.com/pphc"
+              >visit our website </ExternalLink
+            >.
           </p>
           <p>
             <a class="pl-4" href="https://github.com/pph-collective">
```
Comment (mcmcgrath13) [inline src/views/AboutPage.vue:62]: should these have the `tartget="_blank"...` to be consistent - I don't think they need to go through external link, so also feel free to just make an issue for later if you think it should be done eventually

**Review by mcmcgrath13 — APPROVED** [review]
🥇

**Review by eldu — APPROVED** [review]
🔗

**Review by maxwellkrieger — APPROVED** [review]
🥳

**Pull request merged**